data_standards.md

```diff
@@ -51,27 +51,27 @@ To be advised by ACS Data Governance Framework.
 
 #### Directory structure:  
 
-/g/data/ia39/australian-climate-service/\<**status**\>/\<**activity**\>/\<**product**\>/\<**domain**\>/\<**RCM-institution**\>/\<**GCM-model-name**\>/\<**CMIP6-experiment-name**\>/\<**CMIP6-ensemble-member**\>/\<**RCM-model-name**\>/\<**RCM-version-ID**\>/\<**frequency-or-category**\>/\<**variable-name**>
+/g/data/ia39/australian-climate-service/\<**status**\>/\<**activity**\>/\<**product**\>/\<**domain**\>/\<**RCM-institution**\>/\<**GCM-model-name**\>/\<**experiment-name**\>/\<**ensemble-member**\>/\<**RCM-model-name**\>/\<**RCM-version-ID**\>/\<**frequency-or-category**\>/\<**variable-name**>
 
 #### Filenaming:  
 
-\<**variable-name**\>\_\<**domain**\>\_\<**GCM-model-name**\>\_\<**CMIP6-experiment-name**\>\_\<**CMIP6-ensemble-member**\>\_\<**RCM-model-name**\>\_\<**RCM-version-id**\>\_\<**frequency-or-category**\>[\_\<**StartTime-EndTime**\>].nc
+\<**variable-name**\>\_\<**domain**\>\_\<**GCM-model-name**\>\_\<**experiment-name**\>\_\<**ensemble-member**\>\_\<**RCM-model-name**\>\_\<**RCM-version-id**\>\_\<**frequency-or-category**\>[\_\<**StartTime-EndTime**\>].nc
   
 #### Controlled Vocabulary:
 
 | **key** = | [*restriction*:] { item1 item2 ... }  | notes |
 | ------------ | ------------ | --------- |
-| **status** | { `test-data`, `release` }  | |
-| **activity** | { `CORDEX-CMIP6` }  | |
+| **status** | { `test-data`, `staged`, `pre-release`, `release` }  | |
+| **activity** | { `CORDEX-CMIP6`, `ACS-DOWNSCALING-CMIP6`, `ACS-BARRA2` }  | |
```

Comment:
We do need to think about a non-CORDEX activity.  CORDEX-CMIP6 is needed because it needs to fit the CORDEX DRS using this CV item; however I'm not sure what the general 'ACS-DOWNSCALING-CMIP6' will be used for.

I had imagined having 3: 'CORDEX-CMIP6', ACS-CONVECTIVE-CMIP6' and 'ACS-BARRA2'. Would this cover our use cases, or are there others that would require the more general 'downscaling' one?

Reply:
Yes - where does downscaling ERA5 and others fall, it is not part of CMIP6, so I think simply 'ACS-DOWNSCALING' would make sense?

Reply:
Meeting outcome: CORDEX = CORDEX + grid variations